pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/PersistentTransactionBufferProvider.java

```diff
@@ -34,6 +34,7 @@
 /**
  * Persistent transaction buffer provider.
  */
+@Deprecated
```

Comment:
It's some of the old codes are not useful, we can delete them directly. we don't want to tell users there some deprecated components in the first transaction release. 

Please check all.

Reply:
Ok